sql/flow.go

```diff
@@ -0,0 +1,129 @@
+package sql
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path"
+	"runtime"
+	"strings"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/client"
+	"github.com/docker/docker/pkg/archive"
+	"github.com/docker/docker/pkg/stdcopy"
+)
+
+func getPypiVersion() string {
+	url := "https://pypi.org/pypi/astro-sql-cli/json"
+	method := "GET"
+
+	httpClient := &http.Client{}
+	req, err := http.NewRequest(method, url, http.NoBody)
+	if err != nil {
+		fmt.Println(err)
+		return ""
+	}
+	res, err := httpClient.Do(req)
+	if err != nil {
+		fmt.Println(err)
+		return ""
+	}
+	defer res.Body.Close()
+
+	type response struct {
+		Info struct {
+			Version string `json:"version"`
+		} `json:"info"`
+	}
```

Comment:
please move this out of the function logic, it is always better to keep struct definition global